.env.example

```
@@ -107,4 +107,12 @@ CLOUDFLARE_VECTORIZE_INDEX=
 # Used to auto-generate Transistor `transcript_text` for Call Kent episodes.
-CLOUDFLARE_AI_TRANSCRIPTION_MODEL=
\ No newline at end of file
+CLOUDFLARE_AI_TRANSCRIPTION_MODEL=
+
+# Feature: YouTube playlist semantic indexing + /youtube route
+# Optional: set either playlist URL or playlist ID.
+YOUTUBE_PLAYLIST_URL=https://www.youtube.com/watch?v=wSEUlS8WcQs&list=PLV5CVI1eNcJgNqzNwcs4UKrlJdhfDjshf
+YOUTUBE_PLAYLIST_ID=PLV5CVI1eNcJgNqzNwcs4UKrlJdhfDjshf
+# Optional: helps bypass YouTube anti-bot checks for captions in some envs.
+YOUTUBE_COOKIE=
+YOUTUBE_USER_AGENT=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)